client/me/security-2fa-key/index.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons'; // eslint-disable-line no-restricted-imports
+import React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import Security2faKeyAdd from './add';
+import Security2faKeyList from './list';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import { isWebauthnSupported } from 'lib/webauthn';
+import wpcom from 'lib/wp';
+import Notice from 'components/notice';
+
+class Security2faKey extends React.Component {
+	static initialState = Object.freeze( {
```

Comment:
This doesn't seem to be reused. Is there a reason it's extracted and frozen?

Note that freezing here _does not_ prevent the object inside from being mutated, if that was the intention.